examples/Dropdown/Dropdown.md

```diff
@@ -15,6 +15,7 @@
             },
           ]}
           value={value}
+          id="semesters"
```

Comment:
This example should be updated to include a Label

Reply:
Ah, my mistake. I added this through 9bca048. 

And as for your notes above, good points! I'll make sure to include these notes in the ticket for improving the `Dropdown` component.